__tests__/app.test.js

```diff
@@ -224,5 +246,43 @@ describe('Error handling tests', () => {
             })
         })
     })
-})
-    
\ No newline at end of file
+    describe('PATCH `/api/articles/:article_id` errors', () => {
+        test('400: returns status code 400 when sent with an invalid request', () => {
+            return request(app)
+            .patch('/api/articles/test')
+            .send({inc_votes:999})
+            .expect(400)
+            .then(({body: {msg}}) => {
+                expect(msg).toBe('Bad Request')
+            });
+        })
+
+        test('400: returns status code 400 when sent with a valid and existing article id but wrong body property', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .send({test:999})
+            .expect(400)
+            .then(({body: {msg}}) => {
+                expect(msg).toBe('Bad Request')
+            });
+        })
+        test('400: returns status code 400 when sent with a valid and existing article id but with invalid boy properties even if one of them is the required property', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .send({test:999, inc_votes:999})
+            .expect(400)
+            .then(({body: {msg}}) => {
+                expect(msg).toBe('Bad Request')
+            });
+        })
```

Comment:
I would argue that this one would still be a 200 status as it contains the correct key value pair, and should ignore the invalid properties.